src/components/CommentsSection.jsx

```diff
@@ -1,38 +1,124 @@
 import axios from "axios";
 import { useEffect, useState } from "react";
+import { useContext } from "react";
+import { UserContext } from "../contexts/User";
 
-export default ({article_id}) => {
+export default ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [commentBody, setCommentBody] = useState("");
+  const [isEmpty, setIsEmpty] = useState(true);
+  const [disabled, setDisabled] = useState(false);
+  const { user, setUser } = useContext(UserContext);
 
-  const [comments, setComments] = useState([])
+  useEffect(() => {
+    axios
+      .get(
+        `https://nc-news-ngma.onrender.com/api/articles/${article_id}/comments`
+      )
+      .then((response) => {
+        setComments(response.data.comments);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }, [article_id]);
 
-useEffect(()=>{
-  axios.get(`https://nc-news-ngma.onrender.com/api/articles/${article_id}/comments`)
-  .then((response)=>{
-    setComments(response.data.comments)
-  })
-  .catch((err)=>{
-    console.log(err);
-  })
-})
+  const handleAddComment = (e) => {
+    e.preventDefault();
+
+    if (!commentBody) {
```

Comment:
looks like it all works great, but I'm just confused on your naming convention perhaps - you're saying when the comment body is falsy, then isEmpty should be false? Maybe it might be more readable if we called it isFull haha, unless I've misunderstood!

Reply:
I got a bit caught up in things I think. But yes, having taken another look it was a bit misleading. I've refactored.